docs/topics/modelling_considerations.md

```diff
@@ -0,0 +1,194 @@
+could be difficult in this situation.
+
+* It doesn't allow to interact with past events (once again, not easily). If some of the information in the seat could be
+ modified (e.g.: ratings by the audience, or payment receipt information) for past seats, it wouldn't be possible. The same
+ goes for an alarm that was raised multiple times, if some new information arrives about a past alarm notification.
+
+For all those situations where this kind of model cannot fit your problem, you may consider to manage that part of your
+problem externally, through a ticketing system, or a specialized backend. The IoTPlatform offers multiple ways to
+link your data in external systems with data in the platform, so you can still benefit from the Platform features, while
+using specialized tools for particular processes (like support or purchases).
+
+## Attributes are not collections
+
+A commmon mistake when a beginner is modelling a problem in the Platform is: to see the structure of a Context Entity
+(i.e.: objects with an identity that have collections of other objects, the attributes, that have names and values)
+
+      {
+         "type": "SensorMachine",
+         "isPattern": "false",
+         "id": "machine1",
+         "attributes": [
+           {
+             "name": "attr1",
+             "type": "String",
+             "value": "val1"
+           },
+           {
+             "name": "attr2",
+             "type": "String",
+             "value": "val2"
+           },
+           {
+             "name": "attr2",
+             "type": "String",
+             "value": "val2"
+           },
+         ]
+      }
+
+and think that it will fit any model that has an object containing a list of things, like the following one:
+
+      {
+         "type": "DepartmentWorkers",
+         "isPattern": "false",
+         "id": "RRHH",
+         "attributes": [
+           {
+             "name": "Bob",
+             "type": "User",
+             "value": {
+                "name": "Bob",
+                "surname": "Smith",
+                "ID": "3456"
+             }
+           },
+           {
+             "name": "Alice",
+             "type": "User",
+             "value": {
+                "name": "Alice",
+                "surname": "Johnson",
+                "ID": "8744"
+             }
+           }
+         ]
+      }
+
+This is a wrong model for this problem for several reasons:
+
+* First of all, the querying and subscription features of the Context Broker are based on the hypothesis that each of your
+model entities is going to be implemented as an independent Context Entity. Grouping information inside other entities
+would end up in cumbersome queries and subscription logic (and it may even make impossible to execute some queries or
+subscriptions).
+
+* It is also a problem due to performance factors. The performance of some operations depend on the number of attributes
```

Comment:
We could also mention here that there is a physical limit for entities in DB, so ever-growing entity will at the end not storable at CB.